components/scripts/lib/maven.sh

```diff
@@ -52,6 +54,7 @@ invoke_maven() {
 
   args+=(
     -Dmaven.ext.class.path="${extension_classpath}"
+    -Dcom.gradle.enterprise.build_validation.build_number="${build_number}"
```

Comment:
I would rename `build_number` to `buildNumber` (for Gradle and Maven).

Reply:
✅ Done